### DISCUSSION TITLE
教AI透過你的眼睛感知世界

### DISCUSSION THREAD

**Discussion by cobb110202008** [1101-GS3073-C/Python-Community-GS3073-C#65] [科技新知]
最近看到Facebook Artificial Intelligence Research發表了這篇文章，我覺得蠻有趣的，跟大家分享一下
https://ai.facebook.com/blog/teaching-ai-to-perceive-the-world-through-your-eyes/

**Comment by jim11261314** [comment]
我覺得這個計劃跟Meta目前在推的元宇宙，有密切關聯
可以建構一個虛擬的市場，然後再用機器人幫人購物?
增加購物樂趣!
